src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataDeltaFeedSerializer.cs

```diff
@@ -350,7 +363,7 @@ private static IEdmStructuredTypeReference GetResourceType(IEdmTypeReference fee
             throw new SerializationException(message);
         }
 
-        private static Uri GetNestedNextPageLink(ODataSerializerContext writeContext, int pageSize)
+        internal static Uri GetNestedNextPageLink(ODataSerializerContext writeContext, int pageSize)
```

Comment:
>GetNestedNextPageLink [](start = 28, length = 21)

Can we have the same logic as (and maybe even call into) GetNestedNextPageLink on ODataResourceSetSerializer, and trust the skipTokenHandler to deal with the nested delta resource set? or does skiptokenhandler not have the necessary context at that point?